lib/cisco_node_utils/bgp.rb

```diff
@@ -360,6 +360,7 @@ def default_enforce_first_as
     # Nvgen as True With optional 'size <size>
     def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
+      return default_event_history_cli if match == 'size_small'
```

Comment:
You're hard-coding the default value here. Any reason you can't use `default _event_history_cli` instead of `'size_small'`?